test/pylib/util.py

```diff
@@ -33,15 +35,15 @@ def process(self, msg, kwargs):
 T = TypeVar('T')
 
 
-def unique_name():
+def unique_name(unique_name_prefix = 'test_'):
     if not hasattr(unique_name, "last_ms"):
         unique_name.last_ms = 0
     current_ms = int(round(time.time() * 1000))
     # If unique_name() is called twice in the same millisecond...
     if unique_name.last_ms >= current_ms:
         current_ms = unique_name.last_ms + 1
     unique_name.last_ms = current_ms
-    return unique_name_prefix + str(current_ms)
```

Comment:
Looks like we still have a `unique_name_prefix` _global_ variable. How about removing it?

Reply:
yes,thanks